### PR TITLE
Update password reset Documentation (#404)

### LINKED ISSUE #404: Update/replace root password reset doc to v4+ "Built on openSUSE"
Our following page: https://rockstor.com/docs/howtos/reset-root-password.html pertains only to our now legacy CentOS releases prior to V4 "Built on openSUSE". It would be nice to have a V4+ update or alternative and likely maintain the old v3 and earlier doc as a how-to explicitly adding there that it is for the now legacy v3- installs.

Note also that this doc is hard-url referenced from within rockstor-core here:

https://github.com/rockstor/rockstor-core/blob/ccd21f7703d8974282bc05d232efc8f8c67b7700/src/rockstor/storageadmin/templates/storageadmin/login.html#L143

### PR DESCRIPTION
**Pull request by Hooverdan96**
Fixes #404 

### This pull request's proposal
move old password reset procedure to legacy, update existing page with V4 version, as it's also a hard-coded link to the login screen on the Rockstor WebUI.

### Checklist
- [X] With the proposed changes no Sphinx errors or warnings are generated.
- [X] I have added my name to the AUTHORS file, if required (descending alphabetical order).

Note: Link check will continue to fail until currently open PR addressing this is merged.

### CODE REVIEW

File: howtos/reset-root-password.rst [hunk 1]
```diff
@@ -1,81 +1,128 @@
...
+   vbox:~# mount -o bind /dev /mnt/dev
+   vbox:~# chroot /mnt
+   vbox:~# mount -a
 
-Remove autorelabel file ::
+Step 3: Reset password
+----------------------
 
-  bash-4.2# rm -f /.autorelabel
+Now change the password with the *passwd* command ::
```
Comment (phillxnet) [inline howtos/reset-root-password.rst:110]:
```suggestion
Now change the password with the *passwd* command:
```

@Hooverdan96 I think the " ::" formatting directive is clashing with what follows which is the intended console block. In your prior use here the ".. code-block:: console" worked just dandy. But when it follows the similar directive of " ::" it turns out not as intended:

first-console-block-fail

File: howtos/reset-root-password.rst [hunk 2]
```diff
@@ -1,81 +1,128 @@
...
+   vbox:~# passwd
 
-Here's an example image showing execution of above commands.
+Enter the new password. The system will ask for confirmation
+through retyping it. Then the new password is set.
 
-.. image:: /images/howtos/reset-root-password/reset_pw_commands.png
-   :scale: 80%
-   :align: center
+Get out of chroot with the exit command ::
```
Comment (phillxnet) [inline howtos/reset-root-password.rst:119]:
```suggestion
Get out of chroot with the exit command:
```

@Hooverdan96 again I think the prior formatting of " ::" is causing a conflict with the block that follows as we see this when rendered:

second-console-block-fail

**Review by phillxnet — CHANGES_REQUESTED** [review]
@Hooverdan96 Thanks again for yet another update.

This is, as usual, very nice. I've just got a couple of formatting issues that I think, but I'm not certain, are caused by a formatting hang-over. I.e. prior line ending in " ::" clashing with an otherwise defined console block format. Hence my suggestions re using a regular ":" to end the preceding user-visible text as we are introducing that which was promised. And we now have the working earlier in the same new doc `.. code-block:: console`.

I can easily re-test the formatting, post you applying the suggested change at your end, if you have little time to confirm my suspicion regarding these end-format fails.

**Comment by Hooverdan96** [comment]
yes, I think you were right on that. I had fixed some further up before pushing, but I apparently missed a couple (the black background is throwing me off still probably).

**Comment by phillxnet** [comment]
@Hooverdan96 Thanks, and yes that does look to have fixed those console block issues:

console-block-format-fixed

Cheers. I'll merge and get this published then.

**Review by phillxnet — APPROVED** [review]

**Comment by phillxnet** [comment]
PR product PRODUCTION published.

**Pull request merged**